randovania/games/dread/json_data/Elun.txt

```diff
@@ -195,9 +191,13 @@ Extra - asset_id: collision_camera_003
   > Door to Gyroscope Room
       Trivial
   > Event - Elun - Ammo Recharge Room Blob
-      Wave Beam
+      Diffusion Beam or Wave Beam
   > Ammo Recharge
       Trivial
+  > Alcove
+      Diffusion Beam or Wave Beam
+  > Event - Release X
+      Diffusion Beam or Wave Beam
```

Comment:
Remove the connection to the event and change the connection to Alcove to be after the blob. To remove redundancy.